client/components/global-notices/index.jsx

```diff
@@ -77,14 +77,16 @@ const NoticesList = React.createClass( {
 		//needs to be updated.
 		noticesList = noticesList.concat( this.props.storeNotices.map( function( notice, index ) {
 			return (
-				<Notice
-					key={ 'notice-' + index }
-					status={ notice.status }
-					duration = { notice.duration || null }
-					showDismiss={ notice.showDismiss }
-					onDismissClick={ this.props.removeNotice.bind( this, notice.noticeId ) }
-					text={ notice.text }>
-				</Notice>
+				<div className='global-notices__notice'>
```

Comment:
Was this wrapper element needed? It doesn't appear that any specific styles are applied to it.

If necessary, could we achieve the same effect using a modifier class to the rendered `<Notice />` element instead?

In its current form, this causes a warning to be shown in the console because the mapped return value doesn't have a `key`. If kept as a wrapper element, we'd at the very least need to move the `key` prop from the `<Notice />` to the `<div />`.

![warning](https://cloud.githubusercontent.com/assets/1779930/14332164/3e42d442-fc16-11e5-8a81-342076f708e6.png)